flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -71,12 +74,26 @@
 
 	protected abstract B getStateBackend() throws Exception;
 
+	protected abstract long getManagedMemorySize();
+
 	@Rule
 	public final TemporaryFolder tempFolder = new TemporaryFolder();
 
+	@Before
+	public void before() {
+		env = MockEnvironment.builder().setMemorySize(getManagedMemorySize()).build();
```

Comment:
I would suggest to rename `MockEnvironmentBuilder.setMemorySize` into `setManagedMemorySize`.